### ISSUE TITLE
not able to undo branche

### ISSUE THREAD

**Issue by inchl** [theNestruo/z80-asm-meter-vscode#143]
I'm using the negative ts to undo a brance in the code (enabling to select an entire block including the branche to be ignored). However this seems not to be working when a jp is inside a branche. 

```
			; hl = case1 or case2
			jp	(hl)

case1:			jp	nz,test ; [ts=100]
			jp	exit

case2:			; undo case1 brance [ts=-???]  what must ??? be to undo the case1 branche?
			jp	nz,test ; [ts=100]
			jp	exit
```

The jp exit in both branches should have the same timing when being measured by your tool.... But they don't.

EDIT:
Hmm... I used the timing of 'run-until-here'.... but that value I shouldn't use apparently.... if I use an other value (all code being executed) it works......

**Comment by theNestruo** [comment]
Oh, I see what is happening here.

It is not a problem the _timing hint_ not being read. Neither _execution flow_ nor _at exit_ total timings are being calculated because of the following precondition:

> 2. execution flow: When the selection **is a single subroutine (i.e.: there are no unconditional JP, JR or RET instructions in the selection)**, (...).
>
> 3. at exit: When the selection **is a single subroutine** and the selection ends at an exit point (...).

I can add a setting for the next version to ignore this precondition, as it may be useful for this particular scenario... and some other relatively common scenarios I can think of right now.

**Comment by theNestruo** [comment]
Setting to continue beyond unconditional jumps implemented in 5.0.0, available as pre-release/preview here:
https://github.com/theNestruo/z80-asm-meter-vscode/releases/tag/5.0.0-RC1

Please check [v5.x documentation](https://github.com/theNestruo/z80-asm-meter-vscode/blob/version/5.x/README.md) to find related setting(s).

**Comment by theNestruo** [comment]
Released to marketplace (as I did a mistake and already published the 5.0.0 wihtout the _pre-release_ check).

Set this setting to false: `z80-asm-meter.timing.executionFlow.stopOnUnconditionalJump`
